CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Before you contribute to the SPDX License List
+Please familiarize yourself with the SPDX License List and its supporting documentation, so that you understand the pertinent context around the list itself:
+* [Overview of the SPDX License List](https://spdx.org/spdx-license-list/license-list-overview) explains some key background, principles for the inclusion of a license on the SPDX License List, as well as an explanation of the fields contained on the list.
+* [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List. 
+
+# How to contribute/participate: general
+The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done both via the mailing list and bi-weekly calls. In particular, the bi-weekly calls are used to discuss topics and issues that may be difficult to only discuss via email.
+You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX wiki](https://wiki.spdx.org/view/Legal_Team)
+
+## Request a new license or exception be added to the SPDX License List
+The SPDX Legal Team greatly appreciates proposals for new free and open source licenses or exceptions to be added to the SPDX License List.  To request a license or exception be added to the SPDX License List, please:
+
+1.  Review the [license list overview and inclusion principles](https://spdx.org/spdx-license-list/license-list-overview).
+
+2.  Check the historical license and exceptions [tracking page](https://docs.google.com/spreadsheets/d/11AKxLBoN_VXM32OmDTk2hKeYExKzsnPjAVM7rLstQ8s/edit?pli=1#gid=695212681) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure this license or exception has not been previously requested. 
+
+3. Submit your request with all of the following information either via a new Issue or via the SPDX-legal mailing list
+(If you are not on the SPDX-legal mailing list, please join first, see above. If you send your license request without joining the mailing list, this will greatly slow down correspondance in both directions.)
+
+* Issue name or email subject line: "New License/Exception Request: <license/exception name>" 
+* Provide a proposed Full Name for the license or exception.
+* Provide a proposed Short Identifier.
+* Provide a functioning url reference to the license or exception text
```

Comment:
With the issue template approach in #557 [tabled for now][1], and this PR still milestoned for immediate release, I think we should make some changes to the template here to bring it closer to what I've proposed in #557.  For example, I'd like to see this entry split into two: one for an optional URL for the license/exception itself, and one for a required URL showing a project consuming the license/exception.  The language I floated for that is [here][2].  That would avoid issues like we saw in #551, where the URL for the exception was easily provided, but [a number of questions came up][3] once we saw a project attempting to use the exception in practice.  Requiring a single consuming project URL also helps set a (low) minimum bar on our “commonly found” requirement (in flight [here][4]).

[1]: https://github.com/spdx/license-list-XML/pull/557#issuecomment-354327831
[2]: https://github.com/spdx/license-list-XML/pull/557/files#diff-e805b1f70ac9998da695a696fc0b7aefR7
[3]: https://github.com/spdx/license-list-XML/pull/551#discussion_r158852017
[4]: https://github.com/spdx/license-list-XML/pull/538/files#diff-04c6e90faac2675aa89e2176d2eec7d8R2